File: sorting/bubble_sort/bubble_sort.java
public static int [] bubbleSort(int [] array){
    for (int x=0; x<array.length; x++) {
        for(int y=x+1; y<array.length; y++) {
            if(array[x]>array[y]) {
                int temp = array[x];
                array[x] = array[y];
                array[y] = temp;
            }
        }
    }
    return array;
}

File: sorting/bubble_sort/bubble_sort.py
'''Bubble Sort in Python'''

def bubble_sort(alist):
    for i in range(len(alist)-1,0,-1):
        for j in range(i):
            if alist[j]>alist[j+1]:
                temp = alist[j]
                alist[j] = alist[j+1]
                alist[j+1] = temp

if __name__ == '__main__':
    
    #Sample data for sorting.
    ALIST = [54, 26, 93, 17, 77, 31, 44, 55, 20]
    bubble_sort(ALIST)
    print(ALIST)
